libbfio/src/handle.rs

```diff
@@ -45,68 +48,119 @@ impl Handle {
     }
 }
 
-type io_handle_t = *mut IoHandle;
+pub type IoHandleConstRef = *const IoHandle;
+pub type IoHandleRefMut = *mut IoHandle;
+pub type BoxedIoHandleRefMut = *mut *mut IoHandle;
```

Comment:
As I told you, I don't really like those names.
Also, why is `*mut *mut` a `Boxed`? You can do a `&&File::new() as *mut *mut` or something..

Reply:
I've removed them, but I still cant use the bindgen imports unless I'll cast everything to opaque pointer types.